_sass/_custom.scss

```diff
@@ -159,6 +161,10 @@ h2 + .contract-payrolls {
     margin-left: 15px;
   }
 }
+.final-payroll strong {
+  color: $red;
+  font-weight: bold;
+}
 
```

Comment:
This color is already a variable, so we can use `$color-secondary`. See: https://github.com/18F/ogp-payroll/blob/18f-pages/_sass/uswds/core/_variables.scss#L35